Override system library detection to just use *ecl-library-directory*.
The original check fails now that library symlink is removed.

Index: src/cmp/cmpmain.lsp
--- src/cmp/cmpmain.lsp.orig
+++ src/cmp/cmpmain.lsp
@@ -960,12 +960,7 @@ from the C language code.  NIL means \"do not create t
 
 (defun ecl-library-directory ()
   "Finds the directory in which the ECL core library was installed."
-  (cond ((and *ecl-library-directory*
-              (probe-file (merge-pathnames (compile-file-pathname "ecl" :type
-                                            #+dlopen :shared-library
-                                            #-dlopen :static-library)
-                                           *ecl-library-directory*)))
-         *ecl-library-directory*)
+  (cond ((and *ecl-library-directory* *ecl-library-directory*))
         ((probe-file "SYS:BUILD-STAMP")
          (setf *ecl-library-directory* (namestring (translate-logical-pathname "SYS:"))))
         ((error "Unable to find library directory"))))
